src/util/exception_utils.h

```diff
@@ -10,6 +10,11 @@ Author: Fotis Koutoulakis, fotis.koutoulakis@diffblue.com
 #define CPROVER_UTIL_EXCEPTION_UTILS_H
 
 #include <string>
+#include <sstream>
+
+#include <goto-symex/symex_target_equation.h>
```

Comment:
I think this is wrong. This means that 'src/util' now has a dependency on 'goto-symex' which I think is wrong. I also don't see why equation_conversion_exceptiont has been moved into this file, since it's so specific to symex....